src/System.ServiceModel.Primitives/pkg/System.ServiceModel.Primitives.pkgproj

```diff
@@ -28,6 +31,14 @@
       <!-- Supported inbox by all ns1.0-1.2 frameworks -->
       <Value>.NETStandard,Version=v1.0;.NETStandard,Version=v1.1;.NETStandard,Version=v1.2</Value>
     </ValidatePackageSuppression>
+    <!-- Impl shim for System.ServiceModel.dll -->
```

Comment:
same as in UWP, so you will want to put it in both `lib\netstandard2.0` and `ref\netstandard2.0`